.github/workflows/cd.yml

```diff
@@ -19,13 +19,15 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - name: Use default env variables
         run: cp extension/env/common.js.example extension/env/common.js
-      - uses: pnpm/action-setup@v1.2.0
+      - uses: pnpm/action-setup@v1.2.1
         with:
           version: 5.2.6
       - name: Install dependencies
         run: make install
-      - name: Install AWS Cli
-        uses: chrislennon/action-aws-cli@v1.1
+      - id: install-aws-cli
```

Comment:
That's funny, I worked on the same problem today in a different repo: https://github.com/transmute-industries/sidetree.js/pull/121/files

Solved it by using the offical aws action: https://github.com/aws-actions/configure-aws-credentials